taxi/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from taxi_service.settings import AUTH_USER_MODEL
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+    def __str__(self) -> str:
+        return self.name
```

Comment:
Should models.CharField be here as returned type? Because I made the following annotation as written in the checklist file.
![Screenshot from 2023-11-16 17-03-53](https://github.com/mate-academy/py-taxi-service-startproject/assets/130174413/e229d2f2-5baa-4579-ba4f-a6f5e55580b3)